packages/language-server/test/plugins/typescript/features/SemanticTokensProvider.test.ts

```diff
@@ -187,7 +187,7 @@ describe('SemanticTokensProvider', () => {
 
         let index = 0;
         while (index < tokens.length) {
-            result.push(tokens.splice(index, (index += 5)));
```

Comment:
Wrong array method when implementing this. meant to group the token array by 5 so the assertion error can be spotted more easily.